src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+nav {
+  display: flex;
+}
+
+a {
+  padding: 23px 0;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+}
+
+a:not(:first-child) {
+  margin-left: 20px;
+}
+
+a:hover,
+.is-active {
```

Comment:
Usually, you could have many `.is-active` elements. Better to define which `active` element would you like to style up.
```css
nav a.is-active {...}
```
---
```html
<nav class="nav">
  <a class="nav__link">...</a>
</nav>
```
```css
.nav__link.is-active {...}
```